src/oatpp-ssdp/SimpleUdpStreamProvider.cpp

```diff
@@ -61,7 +62,55 @@ SimpleUdpStreamProvider::SimpleUdpStreamProvider(v_uint16 port) : m_port(port),
 
 #if defined(WIN32) || defined(_WIN32)
 
-#error SimpleUdpStreamProvider not implemented for Windows yet. Feel free to implement it and create a PR!
+v_io_handle SimpleUdpStreamProvider::instantiateServer() {
+    oatpp::v_io_handle serverHandle;
+    v_int32 ret;
+    int yes = 1;
+
+    struct addrinfo* result = NULL;
+    struct addrinfo hints;
+
+    memset(&hints, 0, sizeof(hints));
+    hints.ai_family = AF_INET;
+    hints.ai_socktype = SOCK_DGRAM;
+    hints.ai_protocol = IPPROTO_UDP;
+    hints.ai_flags = AI_PASSIVE;
+    auto portStr = oatpp::utils::conversion::int32ToStr(m_port);
+
+    ret = getaddrinfo(NULL, (const char*)portStr->getData(), &hints, &result);
+    if (ret != 0) {
+        OATPP_LOGE("[oatpp::ssdp::SimpleUdpStreamProvider::instantiateServer()]", "Error. Call to getaddrinfo() failed with result=%d: %s", ret, strerror(errno));
+        throw std::runtime_error("[oatpp::ssdp::SimpleUdpStreamProvider::instantiateServer()]: Error. Call to getaddrinfo() failed.");
+    }
+
+    serverHandle = socket(result->ai_family, result->ai_socktype, result->ai_protocol);
+    if (serverHandle < 0) {
+        OATPP_LOGE("[oatpp::ssdp::SimpleUdpStreamProvider::instantiateServer()]", "Error. Couldn't open a socket: socket(%d, %d, %d) %s",
+            result->ai_family, result->ai_socktype, result->ai_protocol, strerror(errno));
+        throw std::runtime_error("[oatpp::ssdp::SimpleUdpStreamProvider::instantiateServer()]: Error. Couldn't open a socket");
+    }
+
+    m_closed = false;
+
+    ret = setsockopt(serverHandle, SOL_SOCKET, SO_REUSEADDR, (const char*)&yes, sizeof(int));
```

Comment:
AFAIK for Windows, one has to set `SO_BROADCAST` for true multi-process compatibility. Without it, the applications would reuse the port, but the already running application won't receive packets anymore.